server/trucks.js

```diff
@@ -248,9 +248,21 @@ var foodTrucks = [
 	}
 ];
 
+
+//look through the schedule for each truck
+// if the schedule contains the name of the day in getDayName
+// include the Truck in the availableTrucks
+function filterByDay(dayName) {
+	 _.filter(foodTrucks, function (truck) {
+          _.any(truck.schedule === dayName);
```

Comment:
I changed the _.any to use a function to check the dayName in truck.schedule array.  I found this confusing.  I also changed the trucks module to return an object with the filterByDay method in it so I could call it in server.js   I thought I should be able to define it like this var filterByDay  = { function filterByDay(dayname) {  underscore stuff } }  Then export.modules = filterByDay.  I couldn't get this through jsHint though so I did it like I did the enhancedDate module.